src/erasure-code/isa/CMakeLists.txt

```diff
@@ -51,7 +51,11 @@ if(HAVE_BETTER_YASM_ELF64)
     ${isal_src_dir}/erasure_code/gf_3vect_mad_avx512.asm
     ${isal_src_dir}/erasure_code/gf_4vect_dot_prod_avx512.asm
     ${isal_src_dir}/erasure_code/gf_4vect_mad_avx512.asm
+    ${isal_src_dir}/erasure_code/gf_5vect_mad_avx512.asm
     ${isal_src_dir}/erasure_code/gf_vect_dot_prod_avx512.asm
+    ${isal_src_dir}/erasure_code/gf_5vect_dot_prod_avx512.asm
```

Comment:
we don't compile isa-l as a whole, but instead cherry-pick from it, so https://github.com/intel/isa-l/blob/master/configure.ac#L243 does not kick in when compiling it. and i cannot find the place where `HAVE_AS_KNOWS_AVX512` is defined. could you point me to it?